File: src/dotnet/Kurdi.ECommerce.Inventory.Core/Entities/Language.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Kurdi.ECommerce.Inventory.Core.Entities
{
    [Table(name: "languages")]
    public class Language
    {
        [Key]
        [Column(name: "language_code")]
        public string LanguageCode { get; set; }
        [Column(name: "language_name")]
        public string LanguageName { get; set; }
        [Column(name: "activation")]
        public bool Activation { get; set; }
        [Column(name: "time_stamps")]
        public TimeStamps TimeStamps { get; set; }

    }
}

File: src/main/java/com/kurdi/inventory/domain/entities/Language.java
package com.kurdi.inventory.domain.entities;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;
import java.io.Serializable;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "languages")
public class Language  implements Serializable {
    @Id
    @Column(name = "language_code")
    String languageCode;
    @Column(name = "language_name")
    String languageName;

}
